woocommerce/woocommerce-ios
# Fix: Reset Activity Log Button to Clear All Log Files

Fixes #12478

## Description
This PR addresses an issue in the "Help" section of the application where the "Reset Activity log" button is not functioning as expected. Upon tapping the button, it only displays an arrow on the right side and fails to clear the current day's log files, only deleting archived logs from previous days. This behavior is misleading as the button suggests it should reset the entire activity log, not just the archived logs.

To resolve this, I have implemented changes to ensure that when the "Reset Activity log" button is pressed, all log files, including those from the current day, are cleared. This aligns with the expected behavior and provides a consistent and intuitive experience for the user.

As part of this PR, I updated also CocoaLumberjack pod to v.3.8.5 

### How to Test

1. Open the application.
2. Navigate to the Menu -> Settings -> Help & Support
3. Tap on "View Application Log."
4. Press the "Reset Activity Log" button.
5. Confirm the popup to the previous screen.
6. Reopen the view, and confirm now you are able to see just a row `Current`.
7. Open the `Current` log. Inside you should be able to see just the message `All logs files erased`.

## Video
(to observe the previous behavior, take a look at this issue https://github.com/woocommerce/woocommerce-ios/pull/12479)
<img src="https://github.com/woocommerce/woocommerce-ios/assets/495617/a32668ac-1b38-427f-aa7f-476b5b8069cb" width=300 />


---
